tardis/plasma/properties/nlte.py

```diff
@@ -14,4 +21,44 @@ class PreviousElectronDensities(PreviousIterationProperty):
     outputs = ('previous_electron_densities',)
 
 class PreviousBetaSobolevs(PreviousIterationProperty):
-    outputs = ('previous_beta_sobolevs',)
\ No newline at end of file
+    outputs = ('previous_beta_sobolevs',)
+
+class HeliumNLTE(ProcessingPlasmaProperty):
+    outputs = ('helium_population',)
+
+    def calculate(self, level_boltzmann_factor, electron_densities,
+        ionization_data, beta_rad, g, g_electron, w, t_rad, t_electrons,
+        delta, zeta_data, number_density):
+        helium_population = deepcopy(level_boltzmann_factor.ix[2])
+        # He I excited states
+        he_one_population = level_boltzmann_factor.ix[2].ix[0] * \
```

Comment:
@aoifeboyle What I meant was. Add functions (maybe static ones) to the other properties in the code that do similar calculations and then both properties use their functions. One such example is in level_populations in line 26 and 47. they both use the same calculation, but it is in the code twice. Use a static function there. We can skype on this if you want.